views/map/index.android.js

```diff
@@ -29,8 +29,7 @@ export default class OlafMapView extends React.Component {
           <Button
             onPress={() => Linking.openURL(mapInfo.url).catch(err => console.error('An error occurred', err))}
             style={styles.button}
-          >
-          See the map!</Button>
+          > View Map </Button>
```

Comment:
The results of that idea: 
![image](https://cloud.githubusercontent.com/assets/12042702/17918453/a8f71e10-698a-11e6-97ad-8bfe14672c7e.png)